lms/djangoapps/grades/subsection_grade.py

```diff
@@ -259,23 +270,14 @@ def update_or_create_model(self, student, score_deleted=False, force_update_subs
 
     def _update_aggregated_scores_from_model(self, model):
```

Comment:
Nittiest of the Nitty: Move `_update_aggregated_scores_from_model` below `bulk_create_models` to follow the top-down convention in the grades code regarding private helpers below public methods.